graphql-java-support/pom.xml

```diff
@@ -70,7 +70,7 @@
                     </execution>
                 </executions>
                 <configuration>
-                    <protocArtifact>com.google.protobuf:protoc:3.9.0:exe:${os.detected.classifier}</protocArtifact>
+                    <protocArtifact>com.google.protobuf:protoc:3.19.4:exe:${os.detected.classifier}</protocArtifact>
```

Comment:
This is not related to Fed2. It's mainly to keep the `protoc` version in sync with the runtime version above and also fixes compiling on Mac M1s